assets/app/views/project.html

```diff
@@ -16,8 +16,14 @@
         <div class="row">
           <div class="col-md-12">
             <div class="tile">
-              <h2 class="service">{{serviceId}}
-                <span class="small"> - routing {{service.spec.protocol}} traffic on {{service.spec.portalIP}}:{{service.spec.port}} to port {{service.spec.containerPort}}</span>
+              <h2 class="service" ng-init="numPorts = service.spec.ports.length">{{serviceId}}
+                <span class="small"> - routing traffic on {{service.spec.portalIP}} -</span>
+                <span class="small" ng-repeat="item in service.spec.ports | orderBy:port | limitTo:2">
```

Comment:
Because this is the service bar in the overview, if it starts wrapping its going to look bad and completely throw off the calculation for the positioning of the little connector.

With the details pane coming in you'll be able to select the service to see all of the ports.

Open to other suggestions?